### ISSUE TITLE
Errors in notes and assignment?

### ISSUE THREAD

**Issue by katpreo** [cortner/math405_2022#4]
Hello Prof. Ortner,

This is Katarina from math 405. I think I have found some typos in L10 and A4. Please have a look! And apologies if I am incorrect.

In L10 when Euler A is introduced I think there is a missing h in the equation for p_(n+1) i.e. an h multiplying the grad V term.

In A4 I think there is a missing negative sign in 4b? i.e. a minus sign in front of the 2i*sin....

Thank you!

Katarina

**Comment by cortner** [comment]
Thanks for flagging this. 
- Regarding A4 : it depends on which definition of the transform you use. It seems I used inconsistent definitions in class and in the assignment. this is definitely bad. I'll think about how to adress it. For now I'll tell the TA to double check for this assignment which definition each individual student is using. 
- Regarding L10 - I assume you mean L11, thank you for spotting that typo. It's fixed and pushed. 

I'll leave this open for now until I've sorted out the A4 problem.
